media/js/mozorg/home/fundraiser-eoy2017.js

```diff
@@ -0,0 +1,31 @@
+(function($) {
+    'use strict';
+
+    var $fundraiser = $('#fundraiser');
+    var storageKey = 'fundraiser-eoy2017';
+    var wasClosed = false;
+
+    // see if visitor previously dismissed the banner
+    try {
+        wasClosed = sessionStorage.getItem(storageKey);
+    } catch(ex) {
+        // fail silently
+    }
+
+    if (!wasClosed) {
```

Comment:
This isn't the fault of this PR, but fwiw I can't get the actual donate.mozilla.org website to work on any version of IE. Edge seems to work fine, as does Chrom / Safari / Firefox. Should we not show the form for any version of IE?

Reply:
Hm, appears to be a TLS issue with IE 9 and 10 (IE 11 works for me). I can get the site to work in IE 9 and 10, but only if I go into the security section and check TLS 1.1.

I [filed a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1422119), but will try to reach out via chat to see if this is something we should restrict here or if MoFo can fix on their end.